test/ProjectService.Mappers.UnitTests/Models/TaskInfoMapperTests.cs

```diff
@@ -101,20 +101,20 @@ public void ShouldThrowArgumentNullExceptionWhenDbTaskIsNull()
             Assert.Throws<ArgumentNullException>(() => _mapper.Map(null, userData, userData));
         }
 
-        [Test]
-        public void ShouldReturnDbTaskSuccessfulWhenUserDataIsNull()
-        {
-            UserData assignedUserData = null;
-            UserData authorData = null;
-
-            _taskInfo.AssignedTo.FirstName = null;
-            _taskInfo.AssignedTo.LastName = null;
-
-            _taskInfo.Author.FirstName = null;
-            _taskInfo.Author.LastName = null;
-
-            SerializerAssert.AreEqual(_taskInfo, _mapper.Map(_dbTask, assignedUserData, authorData));
-        }
+        // [Test]
+        // public void ShouldReturnDbTaskSuccessfulWhenUserDataIsNull()
+        // {
+        //     UserData assignedUserData = null;
+        //     UserData authorData = null;
+        //
+        //     _taskInfo.AssignedTo.FirstName = null;
+        //     _taskInfo.AssignedTo.LastName = null;
+        //
+        //     _taskInfo.Author.FirstName = null;
+        //     _taskInfo.Author.LastName = null;
+        //
+        //     SerializerAssert.AreEqual(_taskInfo, _mapper.Map(_dbTask, assignedUserData, authorData));
+        // }
```

Comment:
remove?

Reply:
Tests are not necessary, mb it can be valuable in future. I thought it is better to leave that delete